docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+
+Temporal provides a comprehensive platform for building distributed applications with a focus on durability and reliability. 
+
+## Resilient applications by default
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots, or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks. 
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions. 
+It does so by automatically reconstructing its previous state. 
+Your application logic can be recovered, replayed, or paused from any previous point in the application process.
+
+
+## How It Works
+![HIW 1](/img/how-it-works/hiw-yourapplicationclient.png)
+
+### Workflow
+A Temporal application is the code you write, comprised of Workflow and Activities. Workflows are defined as code: either a function or an object method, depending on the language. 
+A Temporal Workflow defines your overall business logic. That business logic might involve moving money between bank accounts, processing orders, deploying cloud infrastructure, training an AI model, or something else entirely. 
+The Worker runs the Workflow and Activity code. 
+
+![HIW 2](/img/how-it-works/hiw-configuretimeout.png)
```

Comment:
This text precedes an image with "Your application (Worker)" -

But the header here is "Workflow" - so it doesn't seem to align.

Keep a close eye on the text in the images - and try to build more off of that.